index.html

```diff
@@ -14,9 +14,43 @@
   </head>
   <body>
     <header>
-      
+      <nav>
+        <a id="logo" href="/">My Logo</a>
+        <span id="menu-btn">
+          <svg
+            xmlns="http://www.w3.org/2000/svg"
+            fill="none"
+            viewBox="0 0 24 24"
+            stroke-width="1.5"
+            stroke="currentColor"
+            class="w-6 h-6"
+          >
+            <path
+              stroke-linecap="round"
+              stroke-linejoin="round"
+              d="M3.75 6.75h16.5M3.75 12h16.5m-16.5 5.25h16.5"
+            />
+          </svg>
+        </span>
+      </nav>
```

Comment:
- [ ] Kindly consider making sure that the nav tag contains only the hamburger menu and the `my-logo` outside that will help you to restyle easily on the desktop version